bootstrap.go

```diff
@@ -104,7 +103,7 @@ func bootstrap(s Session) {
 	if err == nil {
 		// We have already finished bootstrap.
 		return
-	} else if !errors2.ErrorEqual(err, errors.ErrDatabaseNotExist) {
+	} else if !terror.Schema.Equal(err, terror.DatabaseNotExists) {
```

Comment:
I have to know which class the error belongs to, if we have many errors and many classes, I don't think it is an easy work to check error equal.